explainaboard/metrics/regression.py

```diff
@@ -0,0 +1,359 @@
+        If so the t-test is applicable, which is much more efficient. Otherwise we do
+        bootstrapping to calculate confidence interval, which is slower and potentially
+        less effective.
+        """
+        return False
+
+    def calc_stats_from_data(
+        self,
+        true_data: list[Union[str, list[str]]],
+        pred_data: list[str],
+        config: Optional[MetricConfig] = None,
+    ) -> MetricStats:
+
+        return MetricStats(
+            np.array(
+                [
+                    (true[0], true[1], true[2], pred)
+                    for true, pred in zip(true_data, pred_data)
+                ]
+            )
+        )
+
+    def aggregate_stats(self, stats: MetricStats) -> np.ndarray:
+        """
+        Aggregate sufficient statistics from multiple examples into a single example
+        :param stats: stats for every example
+        :return: aggregated stats
+        """
+
+        data = stats.get_data()
+        return data
+
+    def get_scores_from_stats(
+        self, agg_stats: np.ndarray, config: Optional[MetricConfig] = None
+    ) -> dict[str, list]:
+        config = cast(SegKtauCorrConfig, self._get_config(config))
+        scores: dict[str, list] = {}
+        for stat in agg_stats:
+            sys_name = stat[0]
+            seg_id = stat[1]
+            manual_score = stat[2]
+            auto_score = stat[3]
+
+            score = float(auto_score) if auto_score != '' else None
+            score_manual = float(manual_score) if manual_score != '' else None
+
+            if config.no_human and (
+                'Human' in sys_name or 'HUMAN' in sys_name or sys_name.startswith('ref')
+            ):
+                continue
+
+            if score_manual is None:
+                continue
+
+            if seg_id not in scores:
+                scores[seg_id] = []
+            scores[seg_id].append([score_manual, score])
+
+        return scores
+
+    def count(self, score: list, config: Optional[MetricConfig] = None):
+        config = cast(SegKtauCorrConfig, self._get_config(config))
+        conc = 0
+        disc = 0
+        num = 0
+        for i in range(1, len(score)):
+            for j in range(0, i):
+                if (
+                    abs(score[i][0] - score[j][0]) < config.threshold
+                    or abs(score[i][0] - score[j][0]) == 0
+                ):
+                    continue
+                elif (
+                    score[i][0] - score[j][0] >= config.threshold
+                ):  # system i is better than system j
+                    if score[i][1] > score[j][1]:
+                        conc += 1
+                    else:
+                        disc += 1
+                    num += 1
+                else:  # system i is worse than system j
+                    if score[i][1] < score[j][1]:
+                        conc += 1
+                    else:
+                        disc += 1
+                    num += 1
```

Comment:
The following suggestion is just to make things a bit more concise:
```suggestion
                if (abs(score[i][0] - score[j][0]) >= config.threshold):
                    manual_better = score[i][0] > score[j][0]
                    system_better = score[i][1] > score[j][1]
                    if manual_better == system_better:
                        conc += 1
                    else:
                        disc += 1
                    num += 1
```